components/cam/src/dynamics/se/interp_mod.F90

```diff
@@ -69,6 +69,11 @@ subroutine setup_history_interpolation(interp_ok, mtapes, interp_output,    &
     type(horiz_coord_t), pointer       :: lon_coord
     character(len=max_hcoordname_len)  :: gridname
 
+    if(any(interp_output)) then
+       if (.not. par%dynproc) &
+       call endrun('setup_history_interpolation: interpolated output not supported if atm_ntasks>dyn_npes')
```

Comment:
I think it will be useful if we print out atm_ntasks and dyn_npes values in the error message so that the user knows what needs to be changed.

Reply:
good idea - will fix.